WEB-108 GSIM/GLIM Overview Not Updated After Activation

JaySoni1 opened the pull request:
**Changes Made :-**

-Fixed group overview to show updated GSIM/GLIM account status and balance after approval and activation.
-Added logic to refresh account data immediately after account actions.

[WEB-108](https://mifosforge.jira.com/jira/software/c/projects/WEB/boards/62?assignee=712020%3A5685dc80-2da8-4d54-80e6-7b69c296926e&selectedIssue=WEB-108)

[WEB-108]: https://mifosforge.jira.com/browse/WEB-108?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

JaySoni1 committed:
WEB-108 GSIM/GLIM Overview Not Updated After Activation

JaySoni1 commented:
@steinwinde Please Review .

steinwinde reviewed (changes requested):
Hi @JaySoni1 , you have left all the AI comments in the code. This would be okay, generally, but when AI writes this, it gives the impression that something is amiss: "This is a placeholder. In a real app, use @ViewChild(GeneralTabComponent) generalTab: GeneralTabComponent;". If this is unnecessary, please remove such comments, which look rather like important FIXME's.

JaySoni1 commented:
@steinwinde Thank you for the feedback. I initially added those comments to keep things systematic after solving the issue, but I understand now that they can be confusing in the final code. I have removed all such comments and updated the PR and will make sure that I will include only necessary comments in the future submission.

steinwinde reviewed (approved):
LGTM

JaySoni1 commented:
@steinwinde Thank You for review
